FIAT/check_format_variant.py

```diff
@@ -0,0 +1,32 @@
+import re
+import warnings
+
+
+def check_format_variant(variant, degree, element):
+    if variant is None:
+        variant = "point"
+        warnings.simplefilter('always', DeprecationWarning)
+        warnings.warn('Variant of ' + element + ' element will change from point evaluation to integral evaluation.'
+                      ' You should project into variant="integral"', DeprecationWarning)
+        # Replace by the following in a month time
+        # variant = "integral"
+
+    if not (variant == "point" or "integral" in variant):
+        raise ValueError('Choose either variant="point" or variant="integral"'
+                         'or variant="integral(Quadrature degree)"')
```

Comment:
This checking is redundant given the else branch later, but this error message is more useful. Perhaps just migrate the error message to the else branch below?